scss/_modal.scss

```diff
@@ -124,14 +124,18 @@
 // Footer (for actions)
 .modal-footer {
   display: flex;
+  flex-wrap: wrap;
   align-items: center; // vertically center
   justify-content: flex-end; // Right align buttons with flex property because text-align doesn't work on flex items
-  padding: $modal-inner-padding;
+  padding: $modal-inner-padding - $modal-footer-margin-between / 2;
   border-top: $modal-footer-border-width solid $modal-footer-border-color;
 
-  // Easily place margin between footer elements
-  > :not(:first-child) { margin-left: .25rem; }
-  > :not(:last-child) { margin-right: .25rem; }
+  // Place margin between footer elements
+  // stylelint-disable selector-max-universal
+  > * {
```

Comment:
This is a no-go for me—we avoid universal selectors like this whenever possible.

Reply:
@mdo Why?

Reply:
Hi @MartijnCuppens this PR explains it #25165

Reply:
I understand that we should use classes as much as possible. But in this case you don't know which elements are going to be placed in the `.modal-footer`.

Also right now the following code is used to place margins between the elements:
``` css
  // Easily place margin between footer elements
  > :not(:first-child) { margin-left: .25rem; }
  > :not(:last-child) { margin-right: .25rem; }
```
Imo this isn't more dogmatic as the use of the universal selector.

Reply:
Oh word, hmm. Those are also universal selectors since they'll match every elements first before limiting to last/first children. Both are bad generally speaking because they take much longer for browsers to parse and apply.

I wonder if a mix of these existing styles and a `.modal-footer-item` would work? Or perhaps we just recommend spacing utils? A single universe selector is better than two though, too. Hmm...

Reply:
With spacing utils, there are some responsive issues (spacing between wrapped elements). Providing the `.modal-footer-item` class would be the best solution if we want to prevent the use of the universal selector. Downside: It **must** be added to all children of the `.modal-footer`.

Reply:
@mdo Have you made your mind up about this?